src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -550,6 +587,27 @@ DiagnosticBag analyze(bool strictAnalysis)
                 {
                     bool badRegion = false;
                     walker.Analyze(ref badRegion, result);
+                    if (walker._implicitlyInitializedFieldsOpt is { } implicitlyInitializedFields)
+                    {
+                        Debug.Assert(walker._requireOutParamsAssigned);
+                        var builder = ArrayBuilder<FieldSymbol>.GetInstance(implicitlyInitializedFields.Count);
+                        // ensure the fields to initialize are ordered deterministically
+                        foreach (var peerMember in member.ContainingType.GetMembers())
+                        {
+                            if (peerMember switch
+                            {
+                                FieldSymbol f => f,
+                                EventSymbol { AssociatedField: FieldSymbol f } => f,
```

Comment:
>AssociatedField: FieldSymbol f [](http://example.com/codeflow?start=46&length=30)

Is the associated field not included in `GetMembers()`?

Reply:
It is not for events. Only for properties. It's a long-standing "quirk" of GetMembers.

Reply:
This is obviated by using LexicalOrderSymbolComparer.